### PR TITLE
Switch to zk connection string

### PR DESCRIPTION
**Pull request by jwroblewski**

### CODE REVIEW

File: zk.go [hunk 1]
```diff
@@ -5,13 +5,16 @@ import (
...
+	"strings"
 	"time"
 
 	"github.com/samuel/go-zookeeper/zk"
 )
 
-// ZK wraps a zookeeper connection
-type ZK struct{ *zk.Conn }
+type ZK struct {
+	conn   *zk.Conn
```
Comment (sixtus) [inline zk.go]: you would save yourself from all the .conn patches if you just leave this nameless. If we want to track upstream, it's probably recommended to do that.

Comment (sixtus) [inline zk.go]: i.e. it's not wrong, just a bigger diff than needed

**Comment by sixtus** [comment]
except for that style comment: :+1:

File: zk.go (hunk 1 above)
Comment (jwroblewski) [inline zk.go]: Reworked here -- https://github.com/liquidm/sarama-cluster/commit/b15651eb105145cddaf6ce1dae0c419f8357f49f

**Pull request merged**